src/app/Applications/Applications.tsx

```diff
@@ -327,8 +328,15 @@ class Applications extends React.Component {
     return [
       {
         title: 'Details',
-        onClick: (event, rowId, rowData, extra) =>
-          window.location.href="/application/"+rowData.cells[0]
+        onClick: (event, rowId, rowData, extra) => {
+          var url = "/application/"+rowData.cells[0]
+          const value=queryString.parse(location.search);
+          const flag=value.debug;
+          if(typeof flag !== "undefined") {
+              url += "?debug="+flag;
+          }
+          window.location.href=url
+        }
```

Comment:
Passing the query parameter here also to support the debug flag from the homepage itself and along with the `Application` webpage.